### PR TITLE
Checking if elvis kicks in.

### PR DESCRIPTION
**Pull request by KuldeepSinh**

### CODE REVIEW

File: src/pre_connect.erl [hunk 1]
```diff
@@ -24,7 +24,7 @@
 compile_packet(Binary) ->
     compile_remaining_length(compile_packet_type(Binary)).
 %%===================================================================
-%% Roughly speaking, compile_packet_type behaves like a compiler-front-end.
+%% Roughly speaking, compile_packet_type/1 behaves like a compiler-front-end.
 %% It scans (creates tokens from the) first byte of the control packet (a packet henceforth), 
 %% And parses (analyzes) the tokens and determines if they are valid values or errors.
```
Comment (elvisci) [inline src/pre_connect.erl]: Line 29 is too long: %% And parses (analyzes) the tokens and determines if they are valid values or errors..

File: src/pre_connect.erl [hunk 2]
```diff
@@ -24,7 +24,7 @@
 compile_packet(Binary) ->
     compile_remaining_length(compile_packet_type(Binary)).
 %%===================================================================
-%% Roughly speaking, compile_packet_type behaves like a compiler-front-end.
+%% Roughly speaking, compile_packet_type/1 behaves like a compiler-front-end.
 %% It scans (creates tokens from the) first byte of the control packet (a packet henceforth),
```
Comment (elvisci) [inline src/pre_connect.erl]: Line 28 is too long: %% It scans (creates tokens from the) first byte of the control packet (a packet henceforth), .

File: src/pre_connect.erl [hunk 3]
```diff
@@ -24,9 +24,9 @@
...
     compile_remaining_length(compile_packet_type(Binary)).
 %%===================================================================
-%% Roughly speaking, compile_packet_type behaves like a compiler-front-end.
-%% It scans (creates tokens from the) first byte of the control packet (a packet henceforth), 
-%% And parses (analyzes) the tokens and determines if they are valid values or errors.
+%% Roughly speaking, compile_packet_type/1 behaves like a compiler-front-end.
+%% It scans (creates tokens from the) first byte of the control packet. 
+%% And parses (analyzes) the tokens and determines their validity.
 %% As an output the function returns...
 %%     values of packet type and bit flags, extracted from the type-byte and remaining binary or
```
Comment (elvisci) [inline src/pre_connect.erl]: Line 31 is too long: %%     values of packet type and bit flags, extracted from the type-byte and remaining binary or.

File: src/pre_connect.erl [hunk 4]
```diff
@@ -24,12 +24,12 @@
...
-%%     error along with its reason and the binary passed to it.
+%% Roughly speaking, compile_packet_type/1 behaves like a compiler-front-end.
+%% It scans (creates tokens from the) first byte of the control packet. 
+%% And parses (analyzes) the tokens and determines their validity.
+%% As an output the function returns following.
+%%  1. Values of packet type and bit flags and remaining binary.
+%%  2. Or error, its reason and the binary passed to it.
 %%===================================================================
 -type packet_binary()     :: <<_:16, _:_*8>>.
 -type packet_type()       :: #packet_type{msgtype::atom(),dup::0 | 1,qos::0 | 1 | 2,retain::0 | 1}.
```
Comment (elvisci) [inline src/pre_connect.erl]: Line 35 is too long: -type packet_type()       :: #packet_type{msgtype::atom(),dup::0 | 1,qos::0 | 1 | 2,retain::0 | 1}..

**Comment by yams-mqtt** [comment]
Elvis for now is disabled.

**Pull request merged**